### PR TITLE
Allow org members to trigger testing in their own org.

### LINKED ISSUE #5793: member of kubernetes-incubator can't trigger tests by "/ok-to-test" in incubator project(s)
See in: https://github.com/kubernetes-incubator/cri-containerd/pull/464
@yanxuean is a member of kubernetes-incubator. When he commented `/ok-to-test`, bot said:
```
@yanxuean: you can't request testing unless you are a kubernetes member.
```
Interesting. Bot checked if he is a member of kubernetes when he tried to trigger tests for PR in kubernetes-incubator project.

Then @mikebrow commented `/ok-to-test`, it worked. (@mikebrow is both member of kubernetes and kubernetes-incubator, I believe the membership of kubernetes took effect).

/cc @cjwagner @Random-Liu @yanxuean

### PR DESCRIPTION
**Pull request by cjwagner**
fixes #5793 

This could increase API token usage a decent amount if lots of non-members try to trigger testing, but I think it will be fine.

/cc @kargakis 
/area prow
FYI @xiangpengzhao

### CODE REVIEW

File: prow/plugins/trigger/trigger.go [hunk 1]
```diff
@@ -115,3 +115,19 @@ func trustedOrgForRepo(config *plugins.Configuration, org, repo string) string {
 	}
 	return org
 }
+
+func isUserTrusted(ghc githubClient, user, trustedOrg, org string) (bool, error) {
+	orgMember, err := ghc.IsMember(trustedOrg, user)
+	if err != nil {
+		return false, err
```
Comment (xiangpengzhao) [inline prow/plugins/trigger/trigger.go]: No matter what err is here, if it errors out earlier, we will miss a chance to check if the user is a member of `org` (they may be).

Comment (kargakis) [reply prow/plugins/trigger/trigger.go]: This is no different from what happened before so I am fine with it.

Comment (xiangpengzhao) [reply prow/plugins/trigger/trigger.go]: :)

**Pull request merged**